### ISSUE TITLE
Please add --delete-conflicting-outputs

### ISSUE THREAD

**Issue by MikeMitterer** [dart-lang/webdev#62]
It's so annoying that webdev always asks me 

```bash
Found 1 declared outputs which already exist on disk. This is likely because the`.dart_tool/build` folder was deleted, or you are submitting generated files to your source repository.
Delete these files?
1 - Delete
2 - Cancel build
3 - List conflicts
```

**Comment by dtaylor-us** [comment]
This issue is a hinderance to me as well.

**Comment by grouma** [comment]
This isn't documented but I believe you can pass the flag as follows `webdev serve -- --delete-conflicting-outputs`. 

We forward along the remaining arguments here:
https://github.com/dart-lang/webdev/blob/382a58901f1ec043499f5aaad3db1f9d8a0a3d9a/webdev/lib/src/command/serve_command.dart#L92

This behavior is included in `2.0.0` which will hopefully be released soon: 
https://github.com/dart-lang/webdev/blob/master/webdev/lib/src/command/serve_command.dart#L111
